src/microbe_stage/MembranePointData.cs

```diff
@@ -105,6 +108,20 @@ public float Radius
         }
     }
 
+    /// <summary>
+    ///   Membrane's height. Should be used as a multiplier for a constant value
```

Comment:
Could you expand the comment a bit here, I'm not fully understanding what the constant value here means?

Reply:
I decided to remove the part about this being used as a multiplier as it makes more sense to immediately calculate true membrane height (and matched it in the code, of course)